app/src/assets/localization/en/drop_tip_wizard.json

```
@@ -1,15 +1,28 @@
 {
   "before_you_begin_do_you_want_to_blowout": "Before you begin, do you need to preserve aspirated liquid?",
+  "blowout_liquid": "Blow out liquid",
```

```suggestion
  "blowout_liquid": "Blow out liquid",
```
I think this line and L4 should be swapped. I'd just resort the JSON.